vignettes/brms_missings.Rmd

```diff
@@ -17,17 +17,20 @@ params:
 ```{r, SETTINGS-knitr, include=FALSE}
 stopifnot(require(knitr))
 options(width = 90)
+knitr::knit_hooks$set(pngquant = knitr::hook_pngquant)
```

Comment:
Since we have loaded knitr at this point already, shall we drop the ´knitr::` here? At least I don't call `knitr::` for `opts_chunk` so its a bit inconsistent.

Reply:
It's a style question of what to do here. I don't mind prefixing knitr to the call here, but I am happy to drop them, sure.